internal/handlers/common/error.go

```diff
@@ -158,7 +171,89 @@ func formatBitwiseOperatorErr(err error, operator string, maskValue any) error {
 	}
 }
 
+// WriteErrors represents slice of protocol write errors.
+type WriteErrors []WriteError
+
+// Error implements error interface.
+func (we *WriteErrors) Error() string {
+	var err string
+	for _, e := range *we {
+		err += e.Error() + ","
+	}
+
+	return err
+}
+
+// Code returns error code.
```

Comment:
Once methods are documented on the interface level, that comment could be just `Code implements XXX interface.`